index.js

```diff
@@ -1,18 +1,16 @@
 import deepEqual from './utils/deepEqual';
 
+let store = {};
 /**
- * {
- *  user: {
- *    value: { id: 123, name: 'Noel' },
- *    subscribers: [
- *      () => {} // The callback function
- *    ]
- *  }
- * }
+ * Manage a variable accross multiple files
+ * The data is associated to a "key" (can be anything)
  */
-let store = {};
-
 export default {
+  /**
+   * Store data into the storage
+   * @param {*} key: the key under wich the value will be stored
```

Comment:
Shouldn't `key` be recommended as `string`?
We don't have any validation there (and I don't think we really need it), but recommending a `string` may be more helpful than `any`